doc/classes/VisualInstance.xml

```diff
@@ -48,8 +53,7 @@
 			<argument index="0" name="base" type="RID">
 			</argument>
 			<description>
-				Sets the base of the VisualInstance, which changes how the engine handles the VisualInstance under the hood.
-				It is recommended to only use [method set_base] if you know what you're doing.
+				Sets the resource that is instanced by this VisualInstance, which changes how the engine handles the VisualInstance under the hood. Equivalent to [method VisualServer.instance_set_base].
```

Comment:
I think it should be "instantiated", but this wrong verb form is pretty common in our API :)